### ISSUE TITLE
Wrong text in comments when doing ETH --> ONE

### ISSUE THREAD

**Issue by JB273** [harmony-one/ethhmy-bridge.frontend#65]
During testing for ETH --> ONE I see this UX glitch:

It says "You will be prompted to sign two transactions", but I only got one prompt!
![image](https://user-images.githubusercontent.com/54506968/104838614-5a7a6a00-58bc-11eb-99a1-3729fd079c64.png)

**Comment by JB273** [comment]
Same is for ONE --> ETH

![image](https://user-images.githubusercontent.com/54506968/104839083-3ff5c000-58bf-11eb-9e71-2ec86b0d94d6.png)

**Comment by gupadhyaya** [comment]
yes the message can be changed to "you will be prompted to sign at most two transactions" @potvik, as approve is not needed for bridging native ONE.
